skyvern/webeye/utils/dom.py

```diff
@@ -249,7 +252,22 @@ async def get_attr(
     async def input_sequentially(
         self, text: str, default_timeout: float = SettingsManager.get_settings().BROWSER_ACTION_TIMEOUT_MS
     ) -> None:
-        await self.locator.press_sequentially(text, timeout=INPUT_TEXT_TIMEOUT)
+        length = len(text)
+        if length > TEXT_PRESS_MAX_LENGTH:
+            # if the text is longer than TEXT_PRESS_MAX_LENGTH characters, we will locator.fill in initial texts until the last TEXT_PRESS_MAX_LENGTH characters
+            # and then type the last TEXT_PRESS_MAX_LENGTH characters with locator.press_sequentially
+            await self.get_locator().fill(text[: length - TEXT_PRESS_MAX_LENGTH])
```

Comment:
why do we need a "get_locator" function instead of using self.locator?

Reply:
For the future design, if we want to add more logic about searching the locator of element, then we can just modify the `get_locator()` function.
For example, change CSS selector to XPath selector or whatever.